### ISSUE TITLE
custom input text

### ISSUE THREAD

**Issue by zeyofu** [LorrinWWW/two-are-better-than-one#4]
Hi, I wonder if there is any code if we want to run a trained model on custom input text, e.g. on a pure txt file?

**Comment by LorrinWWW** [comment]
If the "--lm_emb_path" argument is not a file, the model will try to load it as a _transformers_ checkpoint and generate language model embeddings online (so it can support **tokenized** custom input text -- tokenized words, no need for chars and bert). However, since this functionality was implemented in an earlier time, it does not output attention weights right now. I will look into this and fix it recently.

**Comment by zeyofu** [comment]
Thanks for such quick reply! That's alright I just need the output. Should I also keep --pretrained_wv blank?

**Comment by LorrinWWW** [comment]
Normally you do not need to change anything else. "--pretrained_wv" is the path of GloVe embeddings, if you want to test custom text, you should use the original GloVe embeddings rather than the reduced one (i.e., ./wv/glove.6B.100d.ace05.txt only preserves the words appearing in ACE05). And if we leave "--pretrained_wv" blank, the GloVe embeddings will be disabled.

**Comment by zeyofu** [comment]
Gotcha! So I successfully ran the trained model on custom input, but the result entities and relations are all empty--no entity/relation was found. I used glove 100d, and saved albert large model embedding without head attention for my input text. I wonder if there might be any reason for this output. I tried on the ACE05 again and the accuracy on relation is normal 63%+. I looked into the input embeddings, and found that about half of the tab_embeddings are 0s.
